shared/versions/stack/8.4.asciidoc

```diff
@@ -0,0 +1,57 @@
+:version:                8.4.0
+////
+bare_version never includes -alpha or -beta
+////
+:bare_version:           8.4.0
+:logstash_version:       8.4.0
+:elasticsearch_version:  8.4.0
+:kibana_version:         8.4.0
+:apm_server_version:     8.4.0
+:branch:                 8.4
+:minor-version:          8.4
+:major-version:          8.x
+:prev-major-version:     7.x
+:prev-major-last:        7.17
+:major-version-only:     8
+:ecs_version:            8.4
```

Comment:
@kgeller, please validate that ecs should align with the stack version for the 8.4.0 release.

Reply:
Yes, ECS version will align with the stack version

Reply:
Thank you! <3